core/app.js

```diff
@@ -37,6 +37,12 @@ if(ENV.NODE_TEST === 'true') {
   // e.g., /logmein?as=prabbit
   // see tests/sessions/* for available users
   require('./app_modules/mock-login')(app);
+  var util = require('util');
+  var log_file = fs.createWriteStream(path.join(__dirname, '/../', 'debug.log'), {flags : 'w'});
```

Comment:
Looks great! Instead of the `w` flag, though, should we use `a`? Otherwise the log is destroyed if the server crashes and reboots.